crates/zombie_actor/src/ghost_tracker.rs

```diff
@@ -254,7 +256,7 @@ mod tests {
                 // when the timeout happens the callback should get
                 // the timeout enum in the callback_data
                 match callback_data {
-                    GhostCallbackData::Timeout => me.state = "timed_out".into(),
+                    GhostCallbackData::Timeout(_) => me.state = "timed_out".into(),
```

Comment:
Is this '_' useful data to record somewhere?

Reply:
Yep! I bubbled this up everywhere, except in a couple unit tests like this to keep them simple.